.github/workflows/test-phpstan.yml

```diff
@@ -77,4 +77,4 @@ jobs:
         run: composer update --ansi --no-interaction
 
       - name: Run static analysis
-        run: vendor/bin/phpstan analyse
+        run: vendor/bin/phpstan analyse --xdebug
```

Comment:
Please reset this change

Reply:
No want disable Xdebug with `XDEBUG_MODE=off`?

Reply:
Ah yes. In the Setup PHP step, add `coverage: none`

Reply:
Done